openquake/hazardlib/site.py

```diff
@@ -100,6 +100,13 @@ def __repr__(self):
         return self.__str__()
 
 
+def eq(array1, array2):
+    """
+    Compare two numpy arrays for equality and returns a boolean
+    """
+    return array1.shape == array2.shape and (array1 == array2).all()
```

Comment:
Actually the usage in oq-lite is to check if the hazard and risk site collections are exactly identical, obtained from the same numbers, this is why I did not want to specify a precision. It seems to work for the intended usage.